# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../xine-ui/xine-ui-0.99.6-curl-deprecated-headers.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

# HG changeset patch
# User Darren Salt <linux@youmustbejoking.demon.co.uk>
# Date 1313605081 -3600
# Node ID 6719df8d79c4c6eb9978eb0d30e089cbd7aaacba
# Parent  4d6262fe6f8f7d70aad12b6ff39d98ad784844ea
Don't include <curl/types.h> any more. It's not needed.

diff -r 4d6262fe6f8f -r 6719df8d79c4 src/xitk/download.c
--- a/src/xitk/download.c	Sun Mar 20 15:34:11 2011 +0000
+++ b/src/xitk/download.c	Wed Aug 17 19:18:01 2011 +0100
@@ -28,7 +28,6 @@
 
 #ifdef HAVE_CURL
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 #endif

 
# HG changeset patch
# User Darren Salt <linux@youmustbejoking.demon.co.uk>
# Date 1320785153 0
# Node ID da361fa574e56e1ea5e205530d73d3c4805264d0
# Parent  5c4524d135f9e9f56121867e2724d9bb3fef2b06
No need to include <curl/easy.h> since it's included by <curl/curl.h>.

(Reported by Martin Jacobs.)

diff -r 5c4524d135f9 -r da361fa574e5 src/xitk/download.c
--- a/src/xitk/download.c	Tue Oct 04 17:09:11 2011 +0100
+++ b/src/xitk/download.c	Tue Nov 08 20:45:53 2011 +0000
@@ -28,7 +28,6 @@
 
 #ifdef HAVE_CURL
 #include <curl/curl.h>
-#include <curl/easy.h>
 #endif
 
 #include "common.h"

